gslib/commands/perfdiag.py

```diff
@@ -117,88 +126,92 @@
-              experiments. The default value is 1.
+  -c          Sets the number of `processes
+              <https://en.wikipedia.org/wiki/Process_(computing)>`_ to use
+              while running throughput experiments. The default value is 1.
 
-  -k          Sets the number of threads per process to use while running
-              throughput experiments. Each process will receive an equal number
-              of threads. The default value is 1.
+  -k          Sets the number of `threads
+              <https://en.wikipedia.org/wiki/Thread_(computing)>`_ per process
+              to use while running throughput experiments. Each process will
+              receive an equal number of threads. The default value is 1.
 
               NOTE: All specified threads and processes will be created, but may
-              not by saturated with work if too few objects (specified with -n)
-              and too few components (specified with -y) are specified.
+              not by saturated with work if too few objects (specified with ``-n``)
+              and too few components (specified with ``-y``) are specified.
 
-  -p          Sets the type of parallelism to be used (only applicable when
-              threads or processes are specified and threads * processes > 1).
-              The default is to use fan. Must be one of the following:
+  -p          Sets the type of `parallelism
+              <https://en.wikipedia.org/wiki/Parallel_computing>`_ to be used
+              (only applicable when threads or processes are specified and
+              threads * processes > 1). The default is to use ``fan``. Must be
+              one of the following:
 
               fan
-                 Use one thread per object. This is akin to using gsutil -m cp,
+                 Use one thread per object. This is akin to using gsutil ``-m cp``,
                  with sliced object download / parallel composite upload
                  disabled.
 
               slice
-                 Use Y (specified with -y) threads for each object, transferring
+                 Use Y (specified with ``-y``) threads for each object, transferring
                  one object at a time. This is akin to using parallel object
-                 download / parallel composite upload, without -m. Sliced
+                 download / parallel composite upload, without ``-m``. Sliced
                  uploads not supported for s3.
 
               both
-                 Use Y (specified with -y) threads for each object, transferring
+                 Use Y (specified with ``-y``) threads for each object, transferring
                  multiple objects at a time. This is akin to simultaneously
                  using sliced object download / parallel composite upload and
-                 gsutil -m cp. Sliced uploads not supported for s3.
+                 ``gsutil -m cp``. Sliced uploads not supported for s3.
```

Comment:
Sliced uploads should probably be replaced by "parallel composite uploads" for consistency.